charts/argo-events/values.yaml

```diff
@@ -46,21 +46,21 @@ singleNamespace: true
 sensorController:
   name: sensor-controller
   image: sensor-controller
-  tag: v0.17.0
+  tag: latest
```

Comment:
I don't think we should use `latest` here, otherwise if we update the `latest` image tag, user's deployment will automatically pick up.

Reply:
ok, makes sense